spec/workers/detect_invariants_spec.rb

```diff
@@ -107,5 +107,21 @@
         ),
       )
     end
+
+    it 'detects submitted applications with more than three course choices' do
+      application_form = create(:completed_application_form, submitted_application_choices_count: 4)
```

Comment:
Suggestion: if we create another application form with 3 choices, we can be more certain that the expectation below is testing the behaviour we're after (rather than, eg—simply returning all application forms).